spring-modulith-events/spring-modulith-events-jdbc/src/main/resources/schema-mysql.sql

```diff
@@ -6,5 +6,7 @@ CREATE TABLE IF NOT EXISTS EVENT_PUBLICATION
   SERIALIZED_EVENT VARCHAR(4000) NOT NULL,
   PUBLICATION_DATE TIMESTAMP(6) NOT NULL,
   COMPLETION_DATE  TIMESTAMP(6) DEFAULT NULL NULL,
-  PRIMARY KEY (ID)
-)
+  PRIMARY KEY (ID),
+  INDEX EVENT_PUBLICATION_LISTENER_ID_IDX (LISTENER_ID),
```

Comment:
`SERIALIZED_EVENT` cannot be a part of index key as MySQL accepts only 3072 bytes.